test/regress/interp-try-catch-multi-locals.txt

```diff
@@ -0,0 +1,22 @@
+;;; TOOL: run-interp-spec
+;;; ARGS*: --enable-exceptions
+;;; NOTE: ref: issue-2476
+(module
+  (tag $e0)
+  (func (export "test") (result i32)
+    (local $a i32)
+    (local $b i32)
```

Comment:
Can we just update an existing test instead maybe ?

Reply:
like so?